build.gradle

```diff
@@ -1,5 +1,30 @@
 plugins {
     id 'java'
+    id "com.teamscale" version "34.0.1"
```

Comment:
Where is this addition coming from? 🤔 `step7` also has this.

Reply:
Super strange. Yeah it is there on step7 and I just updated the minor version. GitHub bug?

Reply:
Probably 🤷‍♂️